packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -1878,4 +1878,103 @@ describe('@api-batch-3 RPC Server Acceptance Tests', function () {
       });
     });
   });
+
+  describe('Batch Request Test Suite', async function () {
+    // make sure env is set to true BATCH_REQUESTS_ENABLED
+
+    it('@release Should return a batch of requests', async function () {
```

Comment:
Leave this off for now since we haven't turned it on in production, otherwise `@release` tests against deployed environments will fail

Reply:
No docs but good point we should have it.
Basically we have a lot of tests so we use these string test flags to allow for filtering.
So during release process we run a subset of all the acceptance tests using the `@release` tag